src/mlpack/core/tree/cellbound.hpp

```diff
@@ -89,6 +89,7 @@ class CellBound
   /**
    * Initializes to specified dimensionality with each dimension the empty
    * set.
+   * @param dimension Dimensionality of bound.
```

Comment:
I think there should be an additional `*` containing line above this, just to be consistent.

Reply:
Done. Thanks for pointing it out